src/types/libdef/npm/mozlog_v2.2.x.js

```diff
@@ -0,0 +1,33 @@
+declare module 'mozlog' {
```

Comment:
I contributed it upstream, that's why I've put it in the "npm" directory directly.